tock/employees/models.py

```diff
@@ -180,6 +179,15 @@ def is_late(self):
         else:
             return False
 
+    def display_name(self):
```

Comment:
❤️  handling this in our own 1:1 model -- We can get a little pythony here and use Django's [User.get_full_name](https://github.com/django/django/blob/2.2.17/django/contrib/auth/models.py#L341) to...

This will return any Truthy string returned by `get_full_name` otherwise we'll return the username.

```suggestion
    def display_name(self):
        return self.user.get_full_name() or self.user.username
```

Reply:
@Sgtpluck in case it's confusing, @Jkrzy is suggesting pretty much the same thing I suggested up in the template :-)

Reply:
oh dope! i was trying to figure out what `get_full_name` did; i was trying to figure out if django automagically made getters or something. good to know there are some built-in model methods!